llvm/test/Transforms/LoopVectorize/LoongArch/defaults.ll

```diff
@@ -0,0 +1,65 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt < %s -passes=loop-vectorize -mtriple loongarch64-linux-gnu -mattr=+lasx,+auto-vec -S 2>&1 | FileCheck %s
```

Comment:
remove `2>&1`, this test doesn't check debug output.

Reply:
Thanks!